frontend/public/style/_overrides.scss

```diff
@@ -117,6 +117,11 @@ kbd {
     }
   }
 
+  // `.pf-c-page` specificity required
+  .pf-c-page__header-tools .pf-c-dropdown__toggle {
```

Comment:
This can be combined with the `&__header-tools` declaration above

Reply:
If I combine with `&__header-tools` declaration then the rule will be overridden by [ this broader rule on line 155...](https://github.com/openshift/console/pull/1879/files#diff-307e5a43a73c45e1f3983abc14bcb686R161)

thus I needed it to be scoped under `.pf-c-page .pf-c-page__header-tools .pf-c-dropdown__toggle {font-size: 16px}`

```
// specificity targeting form elements to override --pf-global--FontSize--md
.pf-c-page,
.modal-dialog {
  .pf-c-button,
  .pf-c-dropdown,
  .pf-c-dropdown__menu-item,
  .pf-c-dropdown__toggle,
  .pf-c-form-control {
    font-size: $font-size-base;
    height: auto;
  }
}
```

Reply:
I'd be inclined to go with:

```css
.pf-c-dropdown__toggle {
  font-size: $font-size-pf4-base !important;
}

```
on line 118